### PR TITLE
Remove cucumber browser tests and packages

### PR DESCRIPTION
**Pull request by cwdavies**
Cucumber acceptance tests have been replaced by cypress integration tests

## Removals

- gulp test-acceptance task
- cucumber from documentation
- cucumber feature files and step definitions
- page_objects shared_objects and util that were used by cucumber tests
- cucumber from package.json, yarn.lock, and npm-packages-offline-cache
- selenium-webdriver-4.0.0-alpha and sauce-connect-tunnel-0.2.1

## Changes

- gulp config to remove browser_tests

## How to test this PR

1. gulp test

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Future todos are captured in comments and/or tickets
- [x] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### CODE REVIEW

File: docs/browser-acceptance-tests.md [hunk 1]
```diff
@@ -1,178 +0,0 @@
...
-There are a number of options to the command:
-
- - `gulp lint:build`: Lint only the gulp build scripts.
- - `gulp lint:test`: Lint only the test scripts.
- - `gulp lint:scripts`: Lint only the project source scripts.
- - `--fix`: Add this flag (like `gulp lint --fix` or `gulp lint:build --fix`).
-   to auto-fix some errors, where ESLint has support to do so.
- - `--path`: Add this flag to specify a file to lint,
-   rather than all files. Path is relative to the project root,
-   such as `gulp lint --path=cfgov/unprocessed/js/modules/Analytics.js`.
```
Comment (Scotchester) [inline docs/browser-acceptance-tests.md:178]: We should retain these sections that didn't really correspond to the Cucumber tests. Could you restore them and rename this file to "Other Front-end Testing" (and update the nav link accordingly in `mkdocs.yml`)?

File: frontend.sh [hunk 2]
```diff
@@ -49,7 +49,7 @@ install() {
     echo "Installing frontend development dependencies…"
     yarn install
 
-    # Protractor = JavaScript acceptance testing framework.
+    # Protractor = JavaScript browser testing framework.
```
Comment (Scotchester) [inline frontend.sh:52]: To investigate: Is Protractor needed by Cypress?

**Review by Scotchester — CHANGES_REQUESTED** [review]
Thanks for getting started on this, Craig! I pushed a couple commits that excise a few bits you missed, and have one docs request for you.

File: frontend.sh (hunk 2 above)
Comment (cwdavies) [reply frontend.sh:52]: No Protractor is not needed by Cypress

File: docs/browser-acceptance-tests.md (hunk 1 above)
Comment (cwdavies) [reply docs/browser-acceptance-tests.md:178]: Done

**Pull request merged**